src/utils/dates.js

```diff
@@ -0,0 +1,20 @@
+const DateFormat = {
+  m: ['getMonth', 'setMonth'],
+  d: ['getDate', 'setDate']
+}
+
+export const getTimeIntervalFromToday = (amount, dateFormat) => {
+  const fromDate = new Date()
```

Comment:
@DmitriVanGuard Maybe in this function better to use 'from' and 'to'. As return object has params.
What do you think?

Reply:
@Partysun Sure 👌, updated variables names to `from` and `to`